changelog.d/574.deprecation.rst

```diff
@@ -0,0 +1,11 @@
+The ``cmp`` argument to ``attr.s()`` and ``attr.ib()`` is now deprecated.
+
+Please use ``eq`` to add equality methods (``__eq__`` and ``__ne__``) and ``order`` to add ordering methods (``__lt__``, ``__le__``, ``__gt__``, and ``__ge__``) instead – just like with `dataclasses <https://docs.python.org/3/library/dataclasses.html>`_.
+
+Both are effectively ``True`` by default but it's enough to set ``eq=False`` to disable both at once.
+The passing explicitly ``eq=False, order=True`` will raise a ``ValueError`` though.
```

Comment:
s/The passing explicitly .../Passing \1 explicitly/

Reply:
You didn't catch a group but I think I know what you mean.  🥴

Reply:
:D